checkstyle/checkstyle.xml

```diff
@@ -35,6 +35,9 @@
     <module name="TreeWalker">
         <!-- Coding Checks. See: https://checkstyle.sourceforge.io/config_coding.html -->
         <module name="SimplifyBooleanExpression"/>
+        <module name="EmptyCatchBlock">
```

Comment:
It seems the module should be placed under `Block Checks` section. ([https://checkstyle.sourceforge.io/config_blocks.html](https://checkstyle.sourceforge.io/config_blocks.html))

Reply:
Fixed.